frontend/src/features/properties/components/forms/ParcelDetailForm.tsx

```diff
@@ -201,10 +202,18 @@ const ParcelDetailForm = (props: ParcelPropertyProps) => {
       pidDuplicated = !(await isPidAvailable(values));
     }
 
+    let pinDuplicated = false;
+    if (values.pin && initialValues.pin !== values.pin && values.pin.toString().length < 10) {
```

Comment:
Just thought of the scenario where the person changes the PIN for whatever reason. I guess this would still fire at form level as is.